ONSdigital/census-rm-case-processor
# Remove IAC service and replace with uac-qid service

# Motivation and Context

We have moved the logic to create QID into a new uac-qid service and we are using this service to generate the UACs.

# What has changed



* Removed IAC dispenser
* Removed unique number from uac-qid link entity
* Removed qid creator code
* Created client for the uac-qid service
* Refactored code to use the uac-qid service client 
* Refactored tests

# How to test?



* Run all acceptance tests with docker-dev on the branch add-uac-qid-service and it should pass 


# Links



https://trello.com/c/cXw970Gx/902-build-uac-not-iac-qid-service-8#
